src/core_modules/capture-core/components/WidgetFeedback/WidgetFeedback.types.js

```diff
@@ -1,5 +1,23 @@
 // @flow
 
+export type filteredText = {|
```

Comment:
I suggest we use PascalCase for the type declarations (FilteredText)

Reply:
Sorry, didn't know. Thanks for enlightening me 👍